### PR TITLE
added restoring dock windows

### PR DESCRIPTION
**Pull request by MarsLviv**
added feature to save previous state for dock windows

### CODE REVIEW

File: src/mainwindow/mainwindow.h [hunk 1]
```diff
@@ -33,6 +33,8 @@ class MainWindow : public QMainWindow
 
     void setupMainMenu();
 
+    void SaveState();
+    void RestoreState();
 private slots:
```
Comment (vsakharevych) [inline src/mainwindow/mainwindow.h:45]: Please change methods names according to our code convention (saveState(), restoreState())

Comment (vsakharevych) [inline src/mainwindow/mainwindow.h:45]: Provide some comments to methods saveState / restoreState 
They are not obvious for the others

Comment (MarsLviv) [reply src/mainwindow/mainwindow.h:45]: done

Comment (MarsLviv) [reply src/mainwindow/mainwindow.h:45]: done

**Review by vsakharevych — APPROVED** [review]

**Review by valentynfaychuk — APPROVED** [review]
There are a lot of hardcoded values. Better establish program-wide global variables that describe our organization' and our application' names. Or add class "PairStormData" that holds information about our application, its version, its organization etc.

**Comment by MarsLviv** [comment]
works ass supposed

**Pull request merged**